src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletController.java

```diff
@@ -33,4 +38,12 @@ public ResponseEntity<WalletDTO> createWallet(@Valid @RequestBody CreateWalletDT
 
         return new ResponseEntity<>(walletDTO, headers, HttpStatus.CREATED);
     }
+
+    @DeleteMapping("{id}")
```

Comment:
why haven't you used ``/{id}``? It may be not necessary, but looks much more clear in my opinion.

Reply:
Of course, I have corrected